src/openrct2/ride/Ride.cpp

```diff
@@ -2573,12 +2573,7 @@ bool Ride::CanBreakDown() const
     }
 
     rct_ride_entry* entry = get_ride_entry_by_ride(this);
-    if (entry == nullptr || entry->flags & RIDE_ENTRY_FLAG_CANNOT_BREAK_DOWN)
-    {
-        return false;
-    }
-
-    return true;
+    return !(entry == nullptr || entry->flags & RIDE_ENTRY_FLAG_CANNOT_BREAK_DOWN);
```

Comment:
Ditto.

Reply:
```suggestion
    return entry != nullptr && entry->flags & RIDE_ENTRY_FLAG_CANNOT_BREAK_DOWN;
```

https://en.wikipedia.org/wiki/De_Morgan%27s_laws

Reply:
`return entry != nullptr && (entry->flags & RIDE_ENTRY_FLAG_CANNOT_BREAK_DOWN) == 0;`?